core/templates/pages/contributor-dashboard-page/services/contribution-and-review.service.spec.ts

```diff
@@ -311,11 +313,41 @@ describe('Contribution and review service', () => {
           '2022-01-02'
         )
         .then(response => {
-          expect(response.from_date).toEqual('1 Nov 2022');
-          expect(response.to_date).toEqual('1 Dec 2022');
-          expect(response.contribution_hours).toEqual(1.0);
-          expect(response.team_lead).toEqual('Test User');
-          expect(response.language).toEqual('Hindi');
+          const data = response.certificate_data;
+
+          if (!data) {
```

Comment:
I think what you have is probably fine, but in general we try not to have "if" logic in tests. If the issue is with TypeScript then perhaps this approach (type assertions) might work? https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions

Reply:
Done!